packages/backend/src/database/migrations/20220420075732_cache_compiled_project.ts

```diff
@@ -0,0 +1,23 @@
+import { Knex } from 'knex';
+
+const CACHED_EXPLORES_TABLE_NAME = 'cached_explores';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(CACHED_EXPLORES_TABLE_NAME))) {
+        await knex.schema.createTable(
+            CACHED_EXPLORES_TABLE_NAME,
+            (tableBuilder) => {
+                tableBuilder.specificType(
+                    'cached_explores_id',
```

Comment:
We can simply use the `project_uuid` as the primary key, I think that's fastest

Reply:
Updated 
![image](https://user-images.githubusercontent.com/1983672/164241073-d7ce92c3-b47f-47c1-ab58-1ad6d21e11a7.png)